test/accountlinking-with-session/callcount.test.js

```diff
@@ -154,157 +158,175 @@ describe(`Multi-recipe account linking flows core call counts: ${printPath(
             assert.strictEqual(resp.body.status, "OK");
 
+            let mocked = await getMockedValues();
+            let info = mocked.info;
             assert.strictEqual(info.coreCallCount, 6);
         });
         it("should call the core <=8 times with AL without MFA", async () => {
-            const { app, info } = await setup({
+            await setup({
                 initAccountLinking: true,
                 initMFA: false,
             });
             const email = getTestEmail();
-            const resp = await signUpPOST(app, email);
+            const resp = await signUpPOST(email);
             assert.strictEqual(resp.body.status, "OK");
 
+            let mocked = await getMockedValues();
+            let info = mocked.info;
             assert.strictEqual(info.coreCallCount, 8);
         });
 
         it("should call the core <=12 times with MFA and AL", async () => {
-            const { app, info } = await setup({
+            await setup({
                 initAccountLinking: true,
                 initMFA: true,
             });
             const email = getTestEmail();
-            const resp = await signUpPOST(app, email);
+            const resp = await signUpPOST(email);
             assert.strictEqual(resp.body.status, "OK");
 
+            let mocked = await getMockedValues();
+            let info = mocked.info;
             assert.strictEqual(info.coreCallCount, 12);
         });
     });
 
     describe("sign in", function () {
         it("should call the core <=6 times without MFA or AL", async () => {
-            const { app, info } = await setup({
+            await setup({
                 initAccountLinking: false,
                 initMFA: false,
             });
             const email = getTestEmail();
             await createEmailPasswordUser(email, true);
-            info.coreCallCount = 0;
+            await resetMockedValues();
 
-            const resp = await signInPOST(app, email);
+            const resp = await signInPOST(email);
             assert.strictEqual(resp.body.status, "OK");
 
+            let mocked = await getMockedValues();
+            let info = mocked.info;
             assert.strictEqual(info.coreCallCount, 6);
         });
         it("should call the core <=9 times with AL without MFA", async () => {
-            const { app, info } = await setup({
+            await setup({
                 initAccountLinking: true,
                 initMFA: false,
             });
 
             const email = getTestEmail();
             await createEmailPasswordUser(email);
-            info.coreCallCount = 0;
+            await resetMockedValues();
 
-            const resp = await signInPOST(app, email);
+            const resp = await signInPOST(email);
             assert.strictEqual(resp.body.status, "OK");
 
+            let mocked = await getMockedValues();
+            let info = mocked.info;
             assert.strictEqual(info.coreCallCount, 9);
```

Comment:
that is what I was talking about on our chat, we may migrate the a sort of "store". I can do this change but it will have way more changes in the code. Maybe we can do it on another PR so is easy to check the changes

Reply:
I'm fine with doing these cleanup kind of things in a separate PR. I'll leave this comment open until we another place to track these.